rsyslog/undefined_field/undefined_field.go

```diff
@@ -317,6 +266,24 @@ func replaceDotMoveUndefined(jsonMap map[string]interface{}, topPropLevel bool)
 	return cp, replace_me, has_undefined
 }
 
+func replaceDotMoveUndefinedArray(inputs []interface{}, topPropLevel bool) []interface{} {
```

Comment:
`topPropLevel bool` is unused?

Reply:
Indeed...  I thought "go build" tells me all the unused vars...  Obviously, args are out of the service. :p  Removing it.  Thanks!